arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/AccountSection.java

```diff
@@ -37,11 +36,15 @@ public class AccountSection extends TraceSection implements PostRollbackDefer {
 
   Bytes rawTargetAddress;
   Address targetAddress;
-  AccountSnapshot accountSnapshotBefore;
-  AccountSnapshot accountSnapshotAfter;
+  AccountSnapshot firstAccountSnapshot;
+  AccountSnapshot firstAccountSnapshotNew;
+  AccountSnapshot secondAccountSnapshot;
+  AccountSnapshot secondAccountSnapshotNew;
+  int hubStamp;
 
   public AccountSection(Hub hub) {
     super(hub, maxNumberOfRows(hub));
+    hubStamp = hub.stamp();
```

Comment:
Same old rationalizing:
- the AccountSnapshots are closer to the spec
- we log the `hub.stamp()` for later reverts